HW/feature_eng.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+# user input to receive dataset
+# https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data
+dataset = pd.read_csv(
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data"
+)
+# specifing the response variable
+response_name = " <=50K"
+# Don't include response in other variables, we don't want to plot response vs response.
+col_name = [col for col in dataset.columns if col != response_name]
+
+
+def predictor_response_plots(col_name):
+
+    if (dataset[response_name].dtypes == "object") & (
```

Comment:
This would miss a lot of categorical samples. Because Julien said this homework will only have boolean or continuous samples, you could just do something like if len(set(dataset[response_name])) > 2 to decide if a column is boolean or not. Especially because he mentioned the boolean might just be a 0 or 1 which would look continuous to your code.